src/components/ConfigPanel/style.scss

```diff
@@ -57,6 +77,59 @@
                 height: 1.5rem;
                 border-radius: 8px;
             }
+
+            input:checked + .slider {
+                background-color: var(--themeCol);
+            }
+
+            input[type="range"] {
+                -webkit-appearance: none !important;
+                background: var(--themeCol);
+                height:5px;
+                width: 100%;
+            }
+
+            input[type="range"]::-webkit-slider-thumb {
+                -webkit-appearance: none !important;
```

Comment:
I removed this class because the `appearance: none` was causing problems on chrome (invisible slider thumb)